stdlib/string.gr

```diff
@@ -1452,9 +1572,14 @@ export let forEachCodePointi = (fn: (Number, Number) -> Void, str: String) => {
     // avoid heap allocations. `getCodePoint` will throw
     // MalformedUnicode exception for values exceeding this limit.
     let codePoint = getCodePoint(ptr)
+    Memory.incRef(WasmI32.fromGrain(fn))
```

Comment:
Same question as above. I don't understand this.

Reply:
Same as above